content/gocourse-2019-05-melbourne.slide

```diff
@@ -1142,6 +1142,35 @@ Rob Pike's Talk ([[https://blog.golang.org/toward-go2][Towards Go 2]])
 Dave Cheney Blog ([[https://dave.cheney.net/paste/the-past-present-and-future-of-go2.pdf][Past Present and Future of Go]])
 
 
+* Go for Java devs
+
+- Functions are lambdas
+- Typed constants are enums
```

Comment:
Great point!
Again, an example here might be useful, e.g.:
```go
type ErrCode uint8
const (
	ErrInvalidInput ErrCode = iota
	ErrDuplicate
	ErrNotFound
)
```

Reply:
Done